easybuild/easyconfigs/r/ReFrame-HPC/ReFrame-HPC-2.18.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+
+req_py_majver = 3
+req_py_minver = 5
+
+name = 'ReFrame-HPC'
+version = '2.18'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = """ReFrame is a framework for writing regression tests for HPC systems."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['ba6527f9036d7028d6af8d69e8e74603da239abd056e3b0cebac7ce8290a6e31']
+
+
+options = {'modulename': 'reframe'}
+
+sanity_check_paths = {
+    'files': ['bin/reframe'],
+    'dirs': []
+}
+sanity_check_commands = [
+    ('reframe', '-V'),
+    ('python3', "-c 'from reframe import RegressionTest'")
```

Comment:
You can simply these by just making them regular strings (the tuple format is old and may get deprecated at some point):

```python
sanity_check_commands = [
    "reframe -V",
    "python3 -c 'from reframe import RegressionTest'",
]
```

Rather than having the second one, why not use `options = {'modulename': 'reframe.RegressionTest'}` above?

Reply:
What does the `options` variable do actually? I was wondering.

Reply:
It's a way to specify values for parameters for "extensions" (like Python packages are).

We should have a more direct way, since many "extensions" are now installed stand-alone as well (like here).